doc/migration/version-upgrade-guide.md

```diff
@@ -940,7 +979,20 @@ func serveCRMetrics(cfg *rest.Config, operatorNs string) error {
 
 ### Breaking changes
 
-#### Remove Ansible container sidecar
+#### `TestCtx` in `pkg/test` has been deprecated
+
+ The type name `TestCtx` in `pkg/test` has been deprecated and renamed to `Context`. Users of the e2e framework should: 
+ 
+ - Replace `TestCtx` with `Context`
+ - Replace `NewTestCtx` with `NewContext`
+
+#### Scorecard only supports YAML config files
+  
+The scorecard feature now only supports YAML config files. So, any config file with other extension is deprecated and should be changed for the YAML format. For further information see [`scorecard config file`](https://github.com/operator-framework/operator-sdk/blob/v0.16.x/doc/test-framework/scorecard.md#config-file)
```

Comment:
The scorecard feature now only supports YAML config files. Any config file with other extension is deprecated and should be changed for the YAML format.